python/cudf/cudf/_libxx/null_mask.pyx

```diff
@@ -10,12 +10,22 @@ from cudf._libxx.column cimport Column
 from cudf._libxx.includes.null_mask cimport (
     copy_bitmask as cpp_copy_bitmask,
     create_null_mask as cpp_create_null_mask,
-    bitmask_allocation_size_bytes as cpp_bitmask_allocation_size_bytes
+    bitmask_allocation_size_bytes as cpp_bitmask_allocation_size_bytes,
+    underlying_type_t_mask_state
 )
 
 from cudf.core.buffer import Buffer
 
 
+class MaskState(Enum):
```

Comment:
Should we import alias `Enum` as `PyEnum` here as well for consistency?

Reply:
I ended up having to use `IntEnum`, which circumvents the name collision. I didn't realize I'd end up using that until I was able to compile and run my code. :/